.github/workflows/docker-shared.yml

```diff
@@ -16,6 +16,38 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
+      #
+      # BUILD OR READ FROM CACHE
+      #
+
+      - name: chown /usr/local
+        run: |
+          # See https://github.com/actions/cache/issues/845.
+          sudo chown $(whoami) /usr/local && (chown -R $USER /usr/local/*-musl || true)
+
+      - name: Set up build cache
+        id: build-cache
+        uses: actions/cache@v3
+        with:
+          key: linux-x86_64-cache
+          path: |
+            # # Cache bazel path on Linux.
+            ~/.cache/bazel/_bazel_$(whoami)
+            # Cache musl libc toolchains.
+            /usr/local/*-musl
+
+      - name: Install toolchains
+        if: steps.build-cache.outputs.cache-hit != 'true'
+        run: bazel run //bazel/toolchain:x86_64-linux-musl-gcc
+
+      - name: Build binary
+        if: steps.build-cache.outputs.cache-hit != 'true'
+        run: bazel build :urbit
```

Comment:
This can be removed since the `//docker:push*` targets all depend on `//docker:image_x86_64`, which in turn depends on `//pkg/vere:urbit`.

Reply:
Confirmed, see comment below